internal/compiler-bridge-test/src/test/scala/xsbt/ExtractUsedNamesSpecification.scala

```diff
@@ -38,9 +38,24 @@ class ExtractUsedNamesSpecification extends UnitSpec {
                   |   }
                   |}""".stripMargin
     val compilerForTesting = new ScalaCompilerForUnitTesting
-    val usedNames = compilerForTesting.extractUsedNamesFromSrc(srcA, srcB)
+    val (usedNames, referencedSymbols, definedSymbols) =
+      compilerForTesting.extractUsedNamesFromSrc(srcA, srcB)
     val expectedNames = standardNames ++ Set("a", "c", "A", "B", "C", "D", "b", "X", "BB")
     assert(usedNames("b.X") === expectedNames)
+    val expectedSymbols = Set(
+      (2, 16, "a.A#"),
```

Comment:
I would recommend to come up with a more ergonomic testing solution. In scalameta we use "expect tests" which make it nice to review and easy to create new tests https://blog.janestreet.com/testing-with-expectations/

Here is an example expect file in scalameta repo 
- expect file https://github.com/scalameta/scalameta/blob/master/tests/jvm/src/test/resources/lowlevel.expect#L160-L184
- actual test suite https://github.com/scalameta/scalameta/blob/master/tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala#L44-L47

Reply:
Thanks for the wonderful advice.
I learned a lot.
I will try it soon.